index.html

```diff
@@ -29,5 +29,111 @@ <h4>let's connect</h4>
             <li><a href="#"><img src="./images/Vector3.svg" alt=""></a></li>
         </ul>
     </main>
+
+    <section class="grid-container">
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot1.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot2.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
```

Comment:
[OPTIONAL] Kindly replace the `<section>` tag with `<div>` as described above.

Reply:
Implemented 👍